casepro/cases/tests.py

```diff
@@ -900,6 +904,7 @@ def test_search(self):
             {
                 'id': self.case.pk,
                 'assignee': {'id': self.moh.pk, 'name': "MOH"},
+                'user_assignee': {'id': self.user1.pk, 'name': "Evan"},
                 'contact': {'id': self.ann.pk, 'name': "Ann"},
                 'labels': [{'id': self.aids.pk, 'name': "AIDS"}],
                 'summary': "Summary",
```

Comment:
Addressed in https://github.com/praekelt/casepro/pull/76/commits/e186a6ce54bfc254dc2c2a568b67e2934470ccdf